apps/examples/board_specific_demo/ameba_secure_storage_demo/secure_storage.c

```diff
@@ -59,6 +61,47 @@ static uint32_t SLOT1_START_ADDR = 0x2F000;
 static uint32_t SLOT_SIZE = 0x1000;
 static uint32_t SLOT_AREA_OFFEST = 0x23000;
 
+static int flash_write(uint32_t address, uint32_t len, uint8_t *data) {
+	int ret = -1;
+	FAR uint8_t *buffer;
+	FAR struct mtd_dev_s *dev_mtd = NULL;
+
+	if (address == 0 || len <= 0) {
+		printf("Invalid arguments, offset 0x%x, write size %d\n", address, len);
+		return -1;
+	}
+
+	dev_mtd = up_flashinitialize();
+	if (!dev_mtd) {
+		printf("up_flashinitialize Failed\n");
+		return -1;
+	}
+
+	buffer = (uint8_t *)malloc(MAX_SS_SIZE);
+	if (!buffer) {
+		printf("Failed to allocate buffer %d\n", MAX_SS_SIZE);
+		return -1;
+	}
+	memset(buffer, data, len);
+
+	#ifdef CONFIG_MTD_BYTE_WRITE
+	if (dev_mtd->write != NULL) {
+		ret = MTD_WRITE(dev_mtd, address, len, buffer);
+	}
+	#endif
```

Comment:
As far as  I know, MTD function is not for user context.
@jeongarmy Could you share how to write some data at flash memory?